DESCRIPTION

```diff
@@ -13,7 +13,8 @@ Authors@R: c(person("Garrett", "Grolemund", role = "aut"),
              person("Doug","Mitarotonda", role="ctb"),
              person("Joseph", "Larmarange", role="ctb"),
              person("Jonathan", "Boiser", role="ctb"),
-             person("Chel Hee", "Lee", role = "ctb"))
+             person("Chel Hee", "Lee", role = "ctb"),
+             person("Christopher", "Brown", role = "ctb" ))
```

Comment:
This list includes only most frequent commiters sorted in decreasing order of contributions. The actual number of contributors is very long.